lib/converters/v1.0.0/converter-v1-to-v2.js

```diff
@@ -323,6 +323,20 @@ _.assign(Builders.prototype, {
             data[data.mode] = parseFormData(requestV1.data || requestV1.rawModeData, retainEmpty);
         }
 
+        if (requestV1.dataOptions) {
+            data.options = requestV1.dataOptions;
```

Comment:
```suggestion
            data.options = {};
```
traverse `modes` and set whitelisted options.